### ISSUE TITLE
Some ./configure flags dont work

### ISSUE THREAD

**Issue by dreamcat4** [XthemeOrg/Xtheme#48]
The following ./configure options dont seem to work / have any effect:

``` sh
./configure --datarootdir=/usr/share/xtheme --docdir=/usr/share/doc/xtheme
```

just though you should know.

**Comment by dreamcat4** [comment]
Also:

`/usr/modules`  needs to go in ---> `/usr/share/xtheme/modules` for dist: ubuntu.

Theres no configure option specifically for that one though.

This cmdline was how I built, and the full file listing:

https://gist.github.com/dreamcat4/47c14eabb1c6c69b363b

**Comment by dreamcat4** [comment]
OK, have figured it out now. Needed to add `--enable-fhs-paths` flag...

This cmdline will give a debian/ubuntu style folders output:

```sh
# Xtheme - ./configure && make && make install
RUN git clone https://github.com/XthemeOrg/Xtheme.git \
 && cd Xtheme && git submodule update --init && mkdir out \
 && \
    ./configure --sysconfdir=/etc/xtheme --localstatedir=/var \
    --localedir=/usr/share/locale --datarootdir=/usr/share --prefix=/usr --enable-fhs-paths \
    --enable-contrib --with-pcre --with-perl \
    \
 && make && make install DESTDIR=$PWD/out

# Create linux tarball
RUN cd Xtheme/out && tar -czvf xtheme-linux-x86_64.tar.gz * \
 && cp xtheme-linux-x86_64.tar.gz /out/xtheme/
```

Maybe you could add that to your documentation ^^.

**Comment by siniStar7boy** [comment]
Glad to hear your issue is resolved.  I will consider adding these details to documentation.

**Comment by siniStar7boy** [comment]
(Fixed in commit 31f9979fb24cbed72b48dd5cabd2a41f50445fb2)
